source/DataAccess/DADGenerator.ts

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import IntegrationConfigFactory from '../IntegrationConfigFactory';
+import { IIntegrationConfig, IntegrationType } from '../IIntegrationConfig';
+import { mkdir, writeFile } from 'fs';
+
+export default class DataAccessDoc {
```

Comment:
Since this is producing information that might be tied to a contract we should consider having this covered by some unit/integration tests